### PR TITLE
iris getting started

### PR DESCRIPTION
**Pull request by naturalett**

### CODE REVIEW

**Comment by naturalett** [comment]
@aviemzur @lior-schachter Please review.

**Comment by simlev** [comment]
Can we also add a Troubleshooting section.

**Comment by naturalett** [comment]
@simlev Yes. I will add.

File: docs/README.md [hunk 1]
```diff
@@ -35,10 +35,11 @@ Using simple YAML configuration, create your own schedule data pipelines (a sequ
 perform), application servers,  and more.
 
 ## Getting Started
-A simple getting stated guide for Liminal can be found [here](getting_started.md)
+A simple hello world guide for Liminal can be found [here](getting-started/hello_world.md) \
+A simple Iris classification model guide for Liminal can be found [here](getting-started/iris_classification.md)
```
Comment (liorsav) [inline docs/README.md:39]: A more advanced example which demonstrate a simple data-science workflow can be found [here](getting-started/iris_classification.md

File: docs/getting-started/iris_classification.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+# Getting started / ***Iris Classification***
+
+This guide will allow you to set up your first Apache Liminal environment and allow you to create
+some simple ML pipelines. These will be very similar to the ones you are going to build for real
+production scenarios.
+
+## Prerequisites
+
+Python 3 (3.6 and up)
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:28]: add link as you did for docker desktop

File: docs/getting-started/iris_classification.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+-->
+
+# Getting started / ***Iris Classification***
+
+This guide will allow you to set up your first Apache Liminal environment and allow you to create
+some simple ML pipelines. These will be very similar to the ones you are going to build for real
+production scenarios.
+
+## Prerequisites
+
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:27]: please add virtualenv. you are using it below.

File: docs/getting-started/iris_classification.md [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+Python 3 (3.6 and up)
+
+[Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+*Note: Make sure kubernetes cluster is running in docker desktop (or custom kubernetes installation
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:32]: please delete this - (or custom kubernetes installation on your machine)
its confusing

File: docs/getting-started/iris_classification.md [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+on your machine).*
+
+## Deploying the Example
+
+In this tutorial, we will go through setting up Liminal for the first time on your local machine.
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:37]: remove this. as you already write the same about.

File: docs/getting-started/iris_classification.md [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Getting started / ***Iris Classification***
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:20]: In this tutorial, we will go through setting up Apache Liminal on your local machine and run the first machine-learning workflow example.

File: docs/getting-started/iris_classification.md [hunk 7]
```diff
@@ -0,0 +1,274 @@
...
+*Note: liminal home directory is located in the path defined in LIMINAL_HOME env variable.
+If the LIMINAL_HOME environemnet variable is not defined, home directory defaults to
+~/liminal_home directory.*
+
+Now lets runs liminal:
+```BASH
+liminal start
+```
+The start command spins up the liminal server containers which will run pipelines based on your
+deployed liminal.yml files.
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:96]: The start command spins up 3 containers which will run the workflows defined in the
 liminal.yml file.

File: docs/getting-started/iris_classification.md [hunk 8]
```diff
@@ -0,0 +1,274 @@
...
+It runs the following three containers: 
+* liminal-postgress
+* liminal-webserver
+* liminal-scheduler
+
+Once liminal server has completed starting up, you can navigate to admin UI in your browser:
+[http://localhost:8080](http://localhost:8080)
+By default liminal server starts Apache Airflow servers and admin UI will be that of Apache Airflow.
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:104]: remove this line

File: docs/getting-started/iris_classification.md [hunk 9]
```diff
@@ -0,0 +1,274 @@
...
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Getting started / ***Iris Classification***
+
+This guide will allow you to set up your first Apache Liminal environment and allow you to create
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:22]: remove this paragraph and replace it with the comment above.

File: docs/getting-started/iris_classification.md [hunk 10]
```diff
@@ -0,0 +1,274 @@
...
+*Note: Make sure kubernetes cluster is running in docker desktop (or custom kubernetes installation
+on your machine).*
+
+## Deploying the Example
+
+In this tutorial, we will go through setting up Liminal for the first time on your local machine.
+
+### First, let’s build our example project:
+
+In the dev folder, just clone the example code from liminal:
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:41]: remove 'just'

File: docs/getting-started/iris_classification.md [hunk 11]
```diff
@@ -0,0 +1,274 @@
...
+
+## Prerequisites
+
+Python 3 (3.6 and up)
+
+[Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+*Note: Make sure kubernetes cluster is running in docker desktop (or custom kubernetes installation
+on your machine).*
+
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:34]: Please add this section:

Example flow:
We will define the following steps and services to implement the Iris classification example:
Train, Validate & Deploy - Training and validation execution is managed by Liminal Airflow extension. The training task trains a regression model using a public dataset. We then validate the model and deploy it to a model-store in mounted volume.
Inference - online inference is done using a Python Flask service running on the local Kubernetes in docker desktop. The service exposes the /predict endpoint. It reads the model stored in the mounted drive and uses it to evaluate the request.

File: docs/getting-started/iris_classification.md [hunk 12]
```diff
@@ -0,0 +1,274 @@
...
+Let's build the images you need for the example:
+```BASH
+liminal build
+```
+##### The build will create docker images based on the liminal.yml file in the `images` section.
+
+
+Create a kubernetes local volume:
+```BASH
+liminal create
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:76]: add the mount description - without the yml.

File: docs/getting-started/iris_classification.md [hunk 13]
```diff
@@ -0,0 +1,274 @@
...
+![](../nstatic/iris-classification/airflow_view_dag.png)
+
+
+#### Click on “train” and you will get this popup:
+![](../nstatic/iris-classification/airflow_view_log.png)
+
+
+#### Click on “view log” button and you can see the log of the current task run:
+![](../nstatic/iris-classification/airflow_task_log.png)
+
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:126]: liminal.yml walkthrough
pipelines
....
mount drive definition.

File: docs/getting-started/iris_classification.md [hunk 14]
```diff
@@ -0,0 +1,291 @@
...
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Getting started / ***Iris Classification***
+
+In this tutorial, we will guide you through setting up Apache Liminal on your local machine and run the simple machine-learning workflow example.
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:22]: It should be after the ToC.
REPLACE
In this tutorial, we will guide you through setting up Apache Liminal on your local machine and run the simple machine-learning workflow example.

WITH
In this tutorial, we will guide you through setting up Apache Liminal on your local machine and run a simple machine-learning workflow, based on the classic Iris dataset classification example; more details in this [link] (https://scikit-learn.org/stable/auto_examples/datasets/plot_iris_dataset.html).

File: docs/getting-started/iris_classification.md [hunk 15]
```diff
@@ -0,0 +1,291 @@
...
+
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:24]: Please move it to the top of page.

File: docs/getting-started/iris_classification.md [hunk 16]
```diff
@@ -0,0 +1,291 @@
...
+        mounts:
+          - mount: mymount
+            volume: gettingstartedvol
+            path: /mnt/gettingstartedvol
+```
+###### We specify the `MOUNT_PATH` in which we store the trained model.
+
+## Evaluate the iris classification model
+
+Once the iris classification model is complete, you can launch a pod of the pre-built image which contains a flask server:
```
Comment (liorsav) [inline docs/getting-started/iris_classification.md:199]: Not clear. How to run this part. lets talk.

Comment (liorsav) [inline docs/getting-started/iris_classification.md:199]: Once the iris classification model is complete, you can launch a pod of the pre-built image which contains a flask server:
replace with 
Once the iris classification model training is completed and model is deployed (to the mounted volume), you can launch a pod of the pre-built image which contains a flask server:

**Comment by liorsav** [comment]
LGTM

**Pull request merged**